python/sglang/srt/layers/attention/flashinfer_backend.py

```diff
@@ -30,14 +32,19 @@
         BatchPrefillWithRaggedKVCacheWrapper,
     )
     from flashinfer.cascade import merge_state
-    from flashinfer.decode import _grouped_size_compiled_for_decode_kernels
 
 
 class WrapperDispatch(Enum):
     SLIDING_WINDOW = auto()
     CROSS_ATTENTION = auto()
 
 
+def _grouped_size_compiled_for_decode_kernels(
```

Comment:
With FlashInfer v0.2, Tensor Core will be enabled by default for decoding, so this is no longer necessary.

Reply:
I was looking at this issue: https://github.com/flashinfer-ai/flashinfer/issues/549. I can try running without this function to ensure that it works as intended.